h2/connection.py

```diff
@@ -858,25 +858,35 @@ def send_data(self, stream_id, data, end_stream=False):
         :type data: ``bytes``
         :param end_stream: (optional) Whether this is the last data to be sent
             on the stream. Defaults to ``False``.
+        :param pad_length: (optional) Length of the padding to apply to the
+            data frame. Defaults to ``None`` for no use of padding. Note that
+            a value of ``0`` results in padding of length ``0``
+            (with the "padding" flag set on the frame).
```

Comment:
Please add a `versionadded: 2.6.0` directive to this parameter ([example](https://github.com/apolcyn/hyper-h2/blob/4322904ac68cce2f5271034199cf400a82e289de/h2/connection.py#L294)), so the documentation will reflect when it became available.